charges.py

```diff
@@ -79,21 +95,19 @@ def charge(opportunity):
                     "icon_emoji": ":x:",
                 }
             )
-        return
+
+        raise ChargeException(opportunity, "card error")
 
     except stripe.error.InvalidRequestError as e:
         logging.error(f"Problem: {e}")
-        # TODO should we raise this?
-        return
+        raise ChargeException(opportunity, "invalid request")
     except Exception as e:
         logging.error(f"Problem: {e}")
-        # TODO should we raise this?
-        return
+        raise ChargeException(opportunity, "unknown error")
 
     if card_charge.status != "succeeded":
         logging.error("Charge failed. Check Stripe logs.")
-        # TODO should we raise this?
-        return
+        raise ChargeException(opportunity, "charge failed")
```

Comment:
I think these would be helpful. If we start to feel they add noise to the stripe channel we can always dial back.